service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -196,4 +204,71 @@ public static Document createDocumentWithAttachedFile() {
                 .build())
             .build();
     }
+
+    public static List<Element<C21Order>> createC21Orders() {
+        return ImmutableList.of(
+            Element.<C21Order>builder()
+                .value(C21Order.builder()
+                    .orderTitle("Example Order")
+                    .orderDetails(
+                        "Example order details here - Lorem ipsum dolor sit amet, consectetur adipiscing elit")
+                    .orderDate(DATE_FORMATTER_SERVICE.formatLocalDateTimeBaseUsingFormat(
+                        LocalDateTime.now().plusDays(57), FORMAT_STYLE))
+                    .judgeAndLegalAdvisor(JudgeAndLegalAdvisor.builder()
```

Comment:
It might be of value splitting out building judgeAndLegalAdvisor into it's own method - `createJudgeAndLegalAdvisor`

Reply:
good suggestion, updated